crates/api_models/src/webhooks.rs

```diff
@@ -81,20 +82,33 @@ pub struct IncomingWebhookDetails {
     pub resource_object: Vec<u8>,
 }
 
-#[derive(Debug, Clone, Serialize)]
+#[derive(Debug, Clone, Serialize, ToSchema)]
 pub struct OutgoingWebhook {
+    /// The merchant id of the merchant
     pub merchant_id: String,
+
+    /// The unique event id for each webhook
     pub event_id: String,
+
+    /// The type of event this webhook corresponds to.
+    #[schema(value_type = EventType)]
     pub event_type: api_enums::EventType,
+
+    /// This is specific to the flow, for ex: it will be `PaymentsResponse` for payments flow
     pub content: OutgoingWebhookContent,
     #[serde(default, with = "custom_serde::iso8601")]
+
+    /// The time at which webhook was sent
     pub timestamp: PrimitiveDateTime,
```

Comment:
Don't we need a `serde(with = "...")`?

Reply:
This would be a breaking change for users who are consuming this field, as we are changing the format in which timestamp is sent.